airflow/hooks/mysql_hook.py

```diff
@@ -91,8 +89,7 @@ def get_conn(self):
 
         if conn.extra_dejson.get('charset', False):
             conn_config["charset"] = conn.extra_dejson["charset"]
-            if (conn_config["charset"]).lower() == 'utf8' or\
-                    (conn_config["charset"]).lower() == 'utf-8':
+            if (conn_config["charset"]).lower() == 'utf8' or (conn_config["charset"]).lower() == 'utf-8':
```

Comment:
```suggestion
            if conn_config["charset"].lower() in ('utf8', 'utf-8'):
```
Can we simplify it like that?